components/CardRow.tsx

```diff
@@ -0,0 +1,16 @@
+import { Card, Group } from '@mantine/core'
+import React from 'react'
+
+export const CardRow: React.FC<{
+  children: React.ReactNode
+  rightContent: React.ReactNode
+}> = ({ children, rightContent }) => (
+  <Card p="lg" radius="md" withBorder mb="xs">
```

Comment:
let's not add margins to any components by default. as a general rule of thumb.
margins and spaces between components should be defined by the parent render component -- that one is actually figuring out the proper layout. otherwise those are hard to undo, if no margin is needed
```suggestion
export const CardRow: React.FC<{
  children: React.ReactNode
  rightContent: React.ReactNode,
  ...cardProps
}> = ({ children, rightContent } & CardProps) => (
  <Card p="lg" radius="md" withBorder {...cardProps}>
```

Reply:
will do